econml/iv/dr/_dr.py

```diff
@@ -588,33 +1974,79 @@ def _gen_model_final(self):
         return clone(self.model_final, safe=False)
 
     def _gen_prel_model_effect(self):
-        return _IntentToTreatDRIV(model_Y_X=clone(self.model_Y_X, safe=False),
-                                  model_T_XZ=clone(self.model_T_XZ, safe=False),
+        return _IntentToTreatDRIV(model_y_xw=clone(self.model_y_xw, safe=False),
+                                  model_t_xwz=clone(self.model_t_xwz, safe=False),
                                   prel_model_effect=_DummyCATE(),
                                   model_final=clone(self.flexible_model_effect, safe=False),
-                                  cov_clip=1e-7,
-                                  cv=1,
+                                  featurizer=self._gen_featurizer(),
+                                  fit_cate_intercept=self.fit_cate_intercept,
+                                  cov_clip=self.cov_clip,
```

Comment:
because we are doing re-weigthing, we should set a tiny cov_clip here, e.g. 1e-7.